tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -388,6 +389,8 @@ loaded for each platform and the entry function for the module. It is called ``g
 This file contains a ``configuration`` section that controls the entry function of the module.
 You should also set the minimum compatible Godot version with ``compatability_minimum``,
 which prevents older version of Godot from trying to load your extension.
+The ``reloadable`` flag enables your extension to be automatically loaded every time you recompile it,
+without needing to restart the editor. This only works if you compile your extension in debug mode.
```

Comment:
Maybe mention "loaded by the editor". I had lots of user being confused why "hot reloading" cannot reload in a running game. The term that Godot chose here is rather unfortunate.